docs/Buttons.md

```diff
@@ -0,0 +1,308 @@
+
+### `<BulkDeleteButton>`
+
+Deletes the selected rows. To be used inside [the `<List bulkActionButtons>` prop](./List.md#bulkactionbuttons) (where it's enabled by default).
+
+```jsx
+import * as React from 'react';
+import { Fragment } from 'react';
+import { BulkDeleteButton, BulkExportButton } from 'react-admin';
+
+const PostBulkActionButtons = ({ basePath }) => (
+    <Fragment>
+        <BulkExportButton />
+        <BulkDeleteButton basePath={basePath} />
+    </Fragment>
+);
+
+export const PostList = (props) => (
+    <List {...props} bulkActionButtons={<PostBulkActionButtons />}>
+        ...
+    </List>
+);
+```
+
+![Bulk Delete button](./img/bulk-delete-button.png)
+
+| Prop         | Required | Type            | Default            | Description                         |
+| ------------ | -------- | --------------- | ------------------ | ----------------------------------- |
+| `label`      | Optional | `string`        | 'ra.action.delete' | label or translation message to use |
+| `icon`       | Optional | `React.element` | `<DeleteIcon>`     | iconElement, e.g. `<CommentIcon />` |
+| `exporter`   | Optional | `function`      | -                  | Override the List exporter function |
+| `undoable`   | Optional | `boolean`       | true               | Allow users to cancel the deletion  |
+
+### `<FilterButton>`
+
+This button is an internal component used by react-admin in [the `<Filter>` button/form combo](./List.md#the-filter-buttonform-combo).
+
+![List Filters](./img/list_filter.gif)
+
+### `<SortButton>`
+
+Some List views don't have a natural UI for sorting - e.g. the `<SimpleList>`, or a list of images, don't have column headers like the `<Datagrid>`. For these cases, react-admin offers the `<SortButton>`, which displays a dropdown list of fields that the user can choose to sort on.
+
+![Sort Button](./img/sort-button.gif)
+
+`<SortButton>` expects one prop: `fields`, the list of fields it should allow to sort on. For instance, here is how to offer a button to sort on the `reference`, `sales`, and `stock` fields:
+
+```jsx
+import * as React from 'react';
+import { TopToolbar, SortButton, CreateButton, ExportButton } from 'react-admin';
+
+const ListActions = () => (
+    <TopToolbar>
+        <SortButton fields={['reference', 'sales', 'stock']} />
+        <CreateButton basePath="/products" />
+        <ExportButton />
+    </TopToolbar>
+);
+```
+
+| Prop         | Required | Type            | Default            | Description                         |
+| ------------ | -------- | --------------- | ------------------ | ----------------------------------- |
+| `fields`     | Required | `string[]`      | -                  | List of fields to offer sort on     |
+| `icon`       | Optional | `React.element` | `<DeleteIcon>`     | iconElement, e.g. `<CommentIcon />` |
+| `label`      | Optional | `string`        | 'ra.action.delete' | label or translation message to use |
+
+## Record Buttons
+
+### `<DeleteButton>`
```

Comment:
It seems to lack documentation (or link to other doc) for the 3 buttons here

Reply:
Yes, I chose to not document them now to be able to release the feature not too late. 

I'll work on it "sometime in the future" but not in this PR